aiofcm/connection.py

```diff
@@ -247,6 +288,19 @@ async def send_message(self, message: Message) -> MessageResponse:
                 logger.error('Could not send message %s: %s',
                              message.message_id, e)
 
+    async def maintain_min_connections_open(self):
+        while True:
+            missing_connections = max(0, self.min_connections - len(self.connections))
+            if missing_connections > 0:
+                logger.debug('Creating %d missing connections' % missing_connections)
+                for _ in range(missing_connections):
+                    asyncio.ensure_future(self.acquire())
+
+            await asyncio.sleep(1)
+
+            for connection in self.connections[-missing_connections:]:
```

Comment:
If pool was closed it's better not to refresh connections here.

Reply:
Done